bundles/org.openhab.binding.luxom/src/main/java/org/openhab/binding/luxom/internal/handler/LuxomThingHandler.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luxom.internal.handler;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.luxom.internal.protocol.LuxomAction;
+import org.openhab.binding.luxom.internal.protocol.LuxomCommand;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.ThingStatusInfo;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Base type for all Luxom thing handlers.
+ *
+ * @author Kris Jespers - Initial contribution
+ *
+ */
+@NonNullByDefault
+public abstract class LuxomThingHandler extends BaseThingHandler {
+    private final Logger logger = LoggerFactory.getLogger(LuxomThingHandler.class);
+
+    private String address = "";
+
+    @Override
+    public void initialize() {
+        String id = (String) getThing().getConfiguration().get("address");
+        if (id == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/status.thing-address-missing");
+            address = "noaddress";
+            return;
+        }
+        address = id;
+    }
+
+    public LuxomThingHandler(Thing thing) {
+        super(thing);
+    }
+
+    public abstract void handleCommandCommingFromBridge(LuxomCommand command);
```

Comment:
Could you refactor this method to `handleCommandComingFromBridge` in order to fix the typo?